stock_picking_report_delivery_cost/models/stock_picking.py

```diff
@@ -8,10 +8,10 @@ class StockPicking(models.Model):
 
     # Needed for fields.Monetary
     currency_id = fields.Many2one(
-        related="sale_id.currency_id",
-        readonly=True,
+        comodel_name="res.currency",
         string="Currency",
-        related_sudo=True,  # for avoiding access problems
+        compute="_compute_currency_id",
+        compute_sudo=True,  # for avoiding access problems
```

Comment:
These changes are added in the migration commit, but they should be in the “**[FIX] stock_picking_report_delivery_cost: Fix currency_id field + Improve tests**” commit right?